test/TestSkipInsideYaml.py

```diff
@@ -99,3 +114,12 @@ def test_playbook(default_text_runner):
 def test_role_meta(default_text_runner):
     results = default_text_runner.run_role_meta_main(ROLE_META)
     assert len(results) == 0
+
+
+@pytest.mark.xfail(
+    reason="Bug: https://github.com/ansible/ansible-lint/issues/705",
+    strict=True,
```

Comment:
This is not really necessary because it's already strict by default: https://github.com/ansible/ansible-lint/blob/fd1bb1c/pytest.ini#L63.